generators/kubernetes/templates/_service.yml

```diff
@@ -20,11 +20,19 @@ apiVersion: v1
 kind: Service
```

Comment:
@xetys let not try to split everything now. Lets try to merge this in first following same convention we have now and later we can discuss about splitting or not splitting